client/spec/src/main/asciidoc/index.asciidoc

```diff
@@ -23,16 +23,18 @@
 :version-label!:
 :sectanchors:
 :doctype: book
-:license: Apache License v2.0
+:license: Eclipse Foundation Specification License v1.0
```

Comment:
do we need to change from Apache v2 to Eclipse license?  If so, do we need to also change the headers in the asciidoc files?